src/routes/birthday/create/index.ts

```diff
@@ -1,60 +1,24 @@
 import { container } from '@sapphire/framework';
-import { methods, Route, type ApiRequest, type ApiResponse } from '@sapphire/plugin-api';
+import { methods, Route, type ApiResponse } from '@sapphire/plugin-api';
 import { QueryTypes } from 'sequelize';
-import { ApiVerification } from '../../../helpers/provide/api_verification';
 import { isDateString } from '../../../helpers/utils/date';
+import type { ApiRequest, BirthdayQuery } from '../../../lib/api/types';
+import { authenticated, validateParams } from '../../../lib/api/utils';
 import { APIErrorCode } from '../../../lib/enum/APIErrorCode.enum';
+import { ApplyOptions } from '@sapphire/decorators';
 
+@ApplyOptions<Route.Options>({ route: 'birthday/create' })
```

Comment:
Is it better to use `@ApplyOptions` then populating the `super`?

Reply:
No need to complete the super the function does it automatically with the options we give it 🙂